src/FileStorageServiceProvider.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Yiisoft\Files;
+
+use League\Flysystem\FilesystemAdapter;
+use Yiisoft\Di\Container;
```

Comment:
That requires `yiisoft/di` to be added to `composer.json` as dependency. That doesn't look good for framework-independent package.

Reply:
Ok, you can create a package 'yii-files' and I'll move it there.